.github/workflows/release.yml

```
@@ -0,0 +1,43 @@
+name: Release Devworkspace Operator
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'release version in format 0.y.z'
+        required: true
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
```

should be a fixed version (20.04 or 18.04)
```suggestion
    runs-on: ubuntu-20.04
```